src/components/dev-hub/blog-tag-list.js

```diff
@@ -59,7 +59,7 @@ const BlogTag = ({ children, ...props }) => (
 );
 
 // eslint-disable-next-line no-unused-vars
-const BlogTagList = ({ tags = [], tagLinkBase }) => {
+const BlogTagList = ({ tags = [], tagLinkBase = '/tag' }) => {
```

Comment:
I think the intention was to have urls with the specific type of tag like `language/<languageName>`. Is that information we can retrieve for each tag?

Reply:
Yea let's rework these components a bit to allow on a tag-by-tag basis